src/app/pages/onboarding/welcome/welcome.layout.tsx

```diff
@@ -14,16 +14,14 @@ interface WelcomeLayoutProps {
   onStartOnboarding(): void;
   onRestoreWallet(): void;
 }
-export function WelcomeLayout(props: WelcomeLayoutProps): React.JSX.Element {
-  const {
-    tagline,
-    subheader,
-    isGeneratingWallet,
-    onStartOnboarding,
-    onSelectConnectLedger,
-    onRestoreWallet,
-  } = props;
-
+export function WelcomeLayout({
+  tagline,
+  subheader,
+  isGeneratingWallet,
+  onStartOnboarding,
+  onSelectConnectLedger,
+  onRestoreWallet,
+}: WelcomeLayoutProps): React.JSX.Element {
```

Comment:
@kyranjamie at one point you did not like props to be destructured this way and you made me refactor typically to what was here, has that changed? 🤔 I noticed it is a pref of @pete-watters to refactor these so we should decide on the preferred pattern here.

Reply:
I kind of just do it from habit, sorry! I don't have a major preference but then also I probably shouldn't change things for no reason